packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -320,6 +320,43 @@ class AndroidSdk {
 
   String? getAvdManagerPath() => getCmdlineToolsPath(globals.platform.isWindows ? 'avdmanager.bat' : 'avdmanager');
 
+  String? getNdkCompilerPath(String binaryName) {
```

Comment:
s/getNdkCompilerPath/getNdkBinaryPath

Also, note that the official NDK documentation uses all caps.